kernel/api/src/main/java/org/sakaiproject/util/RequestFilter.java

```diff
@@ -1304,6 +1304,10 @@ protected HttpServletResponse preProcessResponse(Session s, HttpServletRequest r
 			res.setHeader("X-Frame-Options", "SAMEORIGIN");
 		}
 
+		UsageSession us = (UsageSession)s.getAttribute(UsageSessionService.USAGE_SESSION_KEY);
+		if (us != null)
+			res.setHeader("X-Sakai-Session",us.getId());
```

Comment:
Sorry to be annoying on code style, but can you please use brackets like the code above?